geode-gfsh/src/test/resources/expected-pom.xml

```diff
@@ -16,6 +16,11 @@
   See the License for the specific language governing permissions and
   limitations under the License.
   -->
+  <!-- This module was also published with a richer model, Gradle metadata,  -->
+  <!-- which should be used instead. Do not delete the following line which  -->
+  <!-- is to indicate to Gradle or any Gradle module metadata file consumer  -->
+  <!-- that they should prefer consuming it instead. -->
+  <!-- do_not_remove: published-with-gradle-metadata -->
```

Comment:
not totally sure what the implications of adding this are, but I would suggest just leaving this file as it was

Reply:
I do not remember how these lines ended up here. I think it was after adding a dependency to this file that finally was not needed.
I will remove them anyway.

Reply:
Gradle started its own alternative to POM files for other Gradle projects. When Geode moved to Gradle 6.x, we started publishing them to our snapshot repository, and this XML tag in the POM is a bread-crumb to Gradle projects to look-aside for that file.

Problem is, that the Gradle metadata does not handle all our dependency declarations (I do not remember the actual failure at the moment) and downstream projects were having an issue when the pulled the metadata file instead of the POM. So we stopped publishing the metadata until there is a push to restore it.